internal/pkg/runtime/launch/launcher_linux.go

```diff
@@ -1074,6 +1075,11 @@ func (l *Launcher) setCgroups(instanceName string) error {
 		return nil
 	}
 
+	if hidePid {
+		sylog.Debugf("Instance stats will not be available because of hidePid is set")
```

Comment:
Instead of `hidePid is set` use `hidepid option is set on /proc mount`

Reply:
thanks! updated.